actix-web-codegen/Cargo.toml

```diff
@@ -12,7 +12,7 @@ workspace = ".."
 proc-macro = true
 
 [dependencies]
-quote = "^1"
+quote = "=1.0.2"
```

Comment:
The failure has been resolved, could you revert it?

Reply:
Reverted it back